src/components/Posts.jsx

```diff
@@ -0,0 +1,75 @@
+import {useState} from 'react'
+import {FAQ} from '../components/FAQ'
+import {Add_Article} from '../components/Add_Article'
+import {Article} from '../components/Article'
+
+
+const defaultArticles = [{id:1, imgsrc:'./src/assets/images/left.png', alt:"Стол с периферией", 
+title:"Boost your conversion rate", text:"Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto accusantium praesentium eius, ut atque fuga culpa, similique sequi cum eos quis dolorum.",
+date:"Mar 16, 2020 · 6 min read"},
+{id:2, imgsrc:'./src/assets/images/center.png', alt:"Ноутбук на столе", 
+title:"How to use search engine optimization to drive sales", text:"Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto accusantium praesentium eius, ut atque fuga culpa, similique sequi cum eos quis dolorum.",
+date:"Mar 10, 2020 · 4 min read"},
+{id:3, imgsrc:'./src/assets/images/right.png', alt:"Imac на столе", 
+title:"Improve your customer experience", text:"Lorem ipsum dolor sit amet consectetur adipisicing elit. Architecto accusantium praesentium eius, ut atque fuga culpa, similique sequi cum eos quis dolorum.",
+date:"Feb 12, 2020 · 11 min read"}]
+
+export function Posts(props) {
+
+
+    const [grid, setGridView] = useState(true)
+
+    function GridViewHandler(){
+        if(grid){}
+        else{
+            setGridView(true)
+        }
+    }
+    function ListViewHandler(){
+        if(grid){
+            setGridView(false)
+        }
+    }
```

Comment:
а тут так, нельзя сделать?

```suggestion
  function GridViewHandler() {
    setGridView(true)
  }
  function ListViewHandler() {
    setGridView(false)
  }
```